demo/Playground.tsx

```diff
@@ -311,7 +334,7 @@ export const Playground = React.memo<PlaygroundProps>((props) => {
                         toaster={toaster}
                         className={b('editor-view')}
                         stickyToolbar={Boolean(stickyToolbar)}
-                        wysiwygToolbarConfig={wToolbarConfig}
+                        wysiwygToolbarConfig={wysiwygToolbarConfig ?? wToolbarConfig}
```

Comment:
There are only the most basic extensions located in the 'markdown' folder. Let's move GPT to the 'extensions/yfm' directory.